juniper_official/Interfaces/interface-details.yml

```diff
@@ -0,0 +1,30 @@
+---
+intfaceStats:
+  rpc: get-interface-information
+  args:
+    extensive: True
+    interface_name: Null
+  args_key: interface_name
+  item: physical-interface[not(contains(mtu,'Unlimited'))]
```

Comment:
Is MTU mandatory here? if not you can remove.

Reply:
Removed MTU field